package.json

```diff
@@ -55,7 +55,8 @@
       "<rootDir>.*(node_modules)(?!.*clay.*).*$"
     ]
   },
-  "workspaces": [
-    "packages/*"
-  ]
+  "workspaces": {
+    "packages":["packages/*"],
+    "nohoist": ["**/clay-charts-shared"]
```

Comment:
I had to add this so that I could copy over the css files from `charts-shared` to `clay-charts` and `clay-charts-react` on compile. If you hoist this to the mono-repo, there is no way to access those css files at build time.